src/components/Table/AsyncTable/AsyncTable.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+
+import { mockActions } from '../Table.test.helpers';
+
+import AsyncTable from './AsyncTable';
+import MockApiClient from './MockApiClient';
+
+const apiClient = new MockApiClient(100, 500);
+
+describe('Async Table', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.runOnlyPendingTimers();
+    jest.useRealTimers();
+  });
+
+  it('changes filter properties', () => {
+    render(<AsyncTable fetchData={apiClient.getData} actions={mockActions} />);
+    jest.advanceTimersByTime(500);
+    jest.runAllTimers();
```

Comment:
I believe `runAllTimers()` is redundant in this case as the time has already been advanced. If it doesn't work without it, the time might need to be increased by a small bit, `510` or something.